Dockerfile

```diff
@@ -22,6 +23,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     iputils-ping \
     jq \
     lsb-release \
+    vim \
```

Comment:
I don't feel that vim should be required here, we shall better keep it (docker image) small